### PR TITLE
SW-5657 Added deliverable and deliverable edit view to application

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2898?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2898** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2898?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2897** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2897?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2895** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2895?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/DeliverableView/QuestionsDeliverableCard.tsx [hunk 1]
```diff
@@ -62,9 +70,10 @@ const QuestionBox = ({ projectId, variable }: { projectId: number; variable: Var
   );
 };
 
-const QuestionsDeliverableCard = ({ deliverable }: EditProps): JSX.Element | null => {
+const QuestionsDeliverableCard = (props: EditProps): JSX.Element | null => {
   const dispatch = useAppDispatch();
   const theme = useTheme();
+  const { deliverable, hideStatusBadge: hideStatusBadge } = props;
```
Comment (nickgraz) [inline src/components/DeliverableView/QuestionsDeliverableCard.tsx:76]:
```suggestion
  const { deliverable, hideStatusBadge } = props;
```

File: src/components/DeliverableView/QuestionsDeliverableCard.tsx [hunk 2]
```diff
@@ -35,7 +43,7 @@ const QuestionBox = ({ projectId, variable }: { projectId: number; variable: Var
       sx={{ marginBottom: theme.spacing(4) }}
     >
       <Box sx={{ float: 'right', marginBottom: '16px', marginLeft: '16px' }}>
-        <VariableStatusBadge status={firstVariableValueStatus} />
+        {hideStatusBadge !== true && <VariableStatusBadge status={firstVariableValueStatus} />}
```
[the remark this answers is not in the document]
Comment (tommylau523) [reply src/components/DeliverableView/QuestionsDeliverableCard.tsx:46]: Route check would work. My thinking is to keep component behaviors determined by props. Let me know if that makes sense.

[the remark this answers is not in the document]
Comment (nickgraz) [reply src/components/DeliverableView/QuestionsDeliverableCard.tsx:46]: Sounds good to me

**Review by nickgraz — APPROVED** [review]

**Comment by tommylau523** [comment]
### Merge activity

* **Jul 25, 7:03 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2898).

**Pull request merged**